src/render/ScreenCreditController.js

```diff
@@ -51,41 +52,36 @@ define([
          * @classdesc Collects and displays screen credits.
          */
         var ScreenCreditController = function () {
+            RenderableLayer.call(this, "ScreenCreditController");
```

Comment:
RenderableLayer is a very sensible place to start for a Layer subclass. That said, I don't see how we're benefiting from it. We already manage all the renderable, and override doRender.